mri/tests/test_scripts.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+# Package import
+from mri.operators import WaveletN, NonCartesianFFT
+from mri.operators.utils import convert_mask_to_locations
+from mri.reconstructors import SingleChannelReconstructor
+from mri.scripts.gridsearch import launch_grid
+
+# Third party import
+from modopt.math.metrics import ssim
+from modopt.opt.linear import Identity
+from modopt.opt.proximity import SparseThreshold
+import numpy as np
+from pysap.data import get_sample_data
+import unittest
+
+
+class TestScripts(unittest.TestCase):
+    """ Test the scripts in pysap mri like gridsearch
+    """
+    def setUp(self):
+        """ Setup the test variables.
+        """
+        self.n_jobs = 2
+
+    def test_gridsearch_single_channel(self):
+        """Test Gridsearch script in mri.scripts for
+        single channel reconstruction this is a test of sanity
+        and not if the reconstruction is right.
+        """
+        image = get_sample_data('2d-mri')
+        mask = np.ones(image.shape)
+        kspace_loc = convert_mask_to_locations(mask)
+        fourier_op = NonCartesianFFT(samples=kspace_loc, shape=image.shape)
```

Comment:
why did you change to NFFT?

Reply:
The test tests for NFFT, we have an example to test for FFT. I wanted to span both eventually. Given that travis is long, I didnt want to add both here itself.